packages/ui/components/Layout.tsx

```diff
@@ -1,20 +1,19 @@
-import * as React from 'react';
+import { Container } from '@mui/material';
 import Head from 'next/head';
 import Link from 'next/link';
-
-// FIXME: this should be the layout styles
-import styles from '../styles/Home.module.css';
-
-import { Container } from '@mui/material';
+import * as React from 'react';
 
 import Navbar from '../components/Navbar';
+import styles from './Layout.module.css';
 
 // FIXME: needs to extend from base props correctly
 export const Layout = ({
   title,
+  belowNavbarArea, // FIXME: this needs to have a better name
```

Comment:
I would think of this as the `hero` for the page

Reply:
good idea! added you suggestion